### ISSUE TITLE
Audio removed when changing video speed

### ISSUE THREAD

**Issue by niklaspollonen** [selsamman/react-native-transcode#2]
The audio is removed when changing video speed on both Android and IOS. Any way to fix this for android? 

For IOS, I recommend changing to this fork of [SDAVAssetExportSession](https://github.com/kingslay/SDAVAssetExportSession). It allows removing/changing of the pitch correction algorithm. The Audio speed can be easily activated by adding the same scaleTimeRange lines for the audioTrack and removing the !scaleTime from the if statement.

**Comment by selsamman** [comment]
Yes this could more easily be done on IOS where AVFoundation already supports time scaling.  On Android we use the android-transcode library fork.   Adding audio scaling to the android-transcode library is not on the horizon as it is quite complex to do properly.   Having the same behavior on both platforms is a primary goal of this project.

**Comment by niklaspollonen** [comment]
Have you checked on this library as an alternative: [Natario Transcoder](https://github.com/natario1/Transcoder)? Seems like it has audio stretching.

**Comment by selsamman** [comment]
Thanks for that.  It has audio scaling but not transitions like crossfades which we support.  Like the fork we are using it has deviated substantially from the original ypresto version so would not be something easily pulled.  Curious as to your use case for time scaled audio.

**Comment by niklaspollonen** [comment]
yeah, I need to read through the code in both libraries and learn from them.  
I'm working on an audio app(that has video as an extra) so pitch changing is important to make interesting audio. I'm also doing native code to make the video editing faster, but its going a bit slowly on the android side. Therefore a demo is on the works with the tools available.
